src/components/timelapse/DatePicker.vue

```diff
@@ -69,8 +70,14 @@ export default {
         earlyDate = addMinutes(earlyDate, minutesOffset)
         lateDate = addMinutes(lateDate,minutesOffset)
       }
-      eventBus.$emit("dates-selected", earlyDate.toISOString(), lateDate.toISOString())
+      eventBus.$emit("dates-selected", earlyDate, lateDate)
     }
+  },
+  created() {
+    const today = new Date()
+    const yesterday = subDays(today, 1)
+    this.dateTwo = format(today, "YYYY-MM-DD")
+    this.dateOne = format(yesterday, "YYYY-MM-DD")
```

Comment:
It might be a good idea to add `eventBus.$emit("dates-selected", today, yesterday)` after this line and remove the code from lines 127 - 130 in `TimelapseBar.vue`. So in the future, if we decide to change the starting conditions, the dates only need to be changed in one file.